ops/charm.py

```diff
@@ -1431,6 +1499,9 @@ class StorageMeta:
     multiple_range: Optional[Tuple[int, Optional[int]]]
     """Range of numeric qualifiers when multiple storage units are used."""
 
+    properties = List[Literal['transient']]
```

Comment:
"transient" isn't the only value for properties

Reply:
Ah, it was the only one in the docs. Is there a list, or should I just make this an open list of strings?

Reply:
@wallyworld It looks like it's the only one [accepted by the `juju/charm` package](https://github.com/juju/charm/blob/f5d79a50fdf959cd340a853aa65c6af5f06a21b3/meta.go#L1369), too. Regardless, I think it's probably more flexible and future-proof to just make our type annotations a `List[str]`.